dio/lib/src/dio/dio_for_browser.dart

```diff
@@ -29,9 +27,29 @@ class DioForBrowser with DioMixin implements Dio {
     String lengthHeader = Headers.contentLengthHeader,
     Object? data,
     Options? options,
-  }) {
-    throw UnsupportedError(
-      'The download method is not available in the Web environment.',
+  }) async {
+    options ??= DioMixin.checkOptions('GET', options);
+
+    // Set receiveTimeout to 48 hours because `Duration.zero` not work!
```

Comment:
What happened?

Reply:
The download was canceled after 3 seconds. I had to use this method!

Reply:
Please explain why it's not working in a comment. Stating that it doesn't work isn't an actually helpful comment, since the next person working on it will need to figure out why it's not working again.

Reply:
I don't know exactly why this happens, but I get the following error

```dart
*** DioException ***:
uri: https://jsoncompare.org/LearningContainer/SampleFiles/Video/MP4/sample-mp4-file.mp4
DioException [receive timeout]: The request took longer than 0:00:03.000000 to receive data. It was aborted. To get rid of this exception, try raising the RequestOptions.receiveTimeout above the duration of 0:00:03.000000 or improve the response time of the server.
```